internal/gui/time.go

```diff
@@ -18,34 +16,31 @@ func newTimeWindow(g *Gui) (t *timeWindow) {
 		Table: tview.NewTable().SetSelectable(false, false).SetFixed(3,1),
 	}
 
-	t.SetBorder(true)
-	t.setEntries(g)
-	t.setKeybinding(g)
+	t.SetBorder(false)
+	t.setEntries()
+	//t.setKeybinding(g)
 	return
 }
 
 func (t *timeWindow) name() string {
 	return `timeWindow`
 }
 
-func (t *timeWindow) setEntries(g *Gui) {
-	t.entries(g)
+func (t *timeWindow) setEntries() {
 	table := t.Clear()
 
-	for i, stat := range g.state.resources.timeWindow {
-		table.SetCell(i, 0, tview.NewTableCell(stat.Name).
-			SetTextColor(tcell.ColorWhite).
-			SetMaxWidth(30).
-			SetExpansion(2))
+	table.SetCell(0, 0, tview.NewTableCell(`Today`).
+		SetTextColor(tcell.ColorWhite).
+		SetMaxWidth(30).
+		SetExpansion(2))
 
-		table.SetCell(i, 1, tview.NewTableCell(stat.Value).
-			SetTextColor(tcell.ColorWhite).
-			SetMaxWidth(30).
-			SetExpansion(1))
-	}
+	table.SetCell(0, 1, tview.NewTableCell(time.Now().Format(`2006-01-02`)).
+		SetTextColor(tcell.ColorWhite).
+		SetMaxWidth(30).
+		SetExpansion(1))
 }
 
-func (t *timeWindow) setKeybinding(g *Gui) {
+/*func (t *timeWindow) setKeybinding(g *Gui) {
```

Comment:
If all of this is commented out then why not delete it?

Reply:
Actually since the date field (`time.go`) is not being used currently in the layout I will remove the file.